reference/orders.v2.oas2.yml

```diff
@@ -4268,6 +4268,16 @@ components:
         - Empty strings `''` and `null` are invalid for `xxx` and `xxx_customer, and `xxx_merchant`.
         - `xxx`, `xxx_customer`, and `xxx_merchant` default to the value from `catalog` if not supplied in the request.
       type: object
+      x-examples:
+        Product With a Drop Down and a Text Field Modifier:
+          value:
+            product_id: 100
+            quantity: 1
+            product_options:
+              - id: 111
+                value: '123'
```

Comment:
Is '123' a string or integer?

Reply:
In practice for this particular case both string and int will work, but since we are documenting this field as string I will leave it like this.